mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -1831,12 +1812,140 @@ static LogicalResult populateTensorFillLogic(mlir::OpBuilder &b,
   return success();
 }
 
+// Convert the async.launch/async.await pattern back to func.call.
+void undoAsyncLaunchPass(Operation *cloneFunc) {
```

Comment:
New function.

Reply:
Parallel simultaneous evolution.  I'll look at the other pull request and possibly adopt that.